lib/iris/tests/unit/analysis/test_Aggregator.py

```diff
@@ -315,6 +319,32 @@ def test_kwarg_pass_through_combined_kwargs(self):
         expected_kwargs.update(call_kwargs)
         lazy_func.assert_called_once_with(data, axis, **expected_kwargs)
 
+class Test_aggregated_by(tests.IrisTest):
+    def setUp(self):
+        latitude = DimCoord(np.linspace(-90, 90, 4),
+                            standard_name='latitude',
+                            units='degrees')
+        longitude = DimCoord(np.linspace(45, 360, 8),
+                             standard_name='longitude',
+                             units='degrees')
+        time_unit = unit.Unit('hours since epoch',
+                              calendar=unit.CALENDAR_360_DAY)
+        time = DimCoord(np.linspace(104040, 122040, 26),
```

Comment:
The definition of this time coordinate makes it difficult to assess on reading what the first (and subsequent times) are, which in turn makes it hard to tell what you are expecting the test result to be and why. Can you change it to something more readable like `days since ...` and have the first time coordinate value be `0` so it is clear what the coordinate looks like?